files/usr/sbin/install-nx.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
```

Comment:
Remember to always add the license header https://nethserver.github.io/nextsecurity/license_headers/.
```suggestion
#!/bin/bash
#
# Copyright (C) 2022 Nethesis S.r.l.
# SPDX-License-Identifier: GPL-2.0-only
#
```